client/my-sites/checkout/cart/cart-buttons.jsx

```diff
@@ -6,71 +6,47 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import createReactClass from 'create-react-class';
-import { identity, isFunction } from 'lodash';
+import { connect } from 'react-redux';
+import { identity } from 'lodash';
 import page from 'page';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import AnalyticsMixin from 'lib/mixins/analytics';
+import Button from 'components/button';
+import { recordGoogleEvent } from 'state/analytics/actions';
 
-export const CartButtons = createReactClass( {
-	displayName: 'CartButtons',
-	mixins: [ AnalyticsMixin( 'popupCart' ) ],
-
-	propTypes: {
+export class CartButtons extends React.Component {
```

Comment:
nit: there's only a single button in that component so I wonder whether we should rename this component to a singular case.

Reply:
The component is a button toolbar, and it happens to have just one button right now. I'd rather keep the name as it is. The possibility to add another button is still open.

Reply:
fine by me 👍